aten/src/ATen/Context.cpp

```diff
@@ -149,6 +149,14 @@ bool Context::isXNNPACKAvailable() const {
 #endif
 }
 
+bool Context::releaseOriginalWeights() const {
```

Comment:
Also can we chose a different name? Since as of now this applies specifically to qnnpack maybe we should call it out such. Like `releaseQNNPACKOriginalWeights`. Not a great name perhaps but the current one is too broad.

Reply:
Sure, I thought you may need to use the same flag for XNNPACK as well. If that is not the case then I will make it QNNPACK specific. let me know

Reply:
How about `setReleaseWeightsWhenPrepacking`?